client/blocks/presence-button/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { isNull, noop, omitBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import { getPostPresenceCount } from 'state/presence/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function PresenceButton( props ) {
+	const { presenceCount, href, onClick, showLabel, tagName, target } = props;
+	const translate = useTranslate();
+
+	return React.createElement(
+		tagName,
+		omitBy(
+			{
+				className: 'presence-button',
+				href: 'a' === tagName ? href : null,
+				onClick,
+				target: 'a' === tagName ? target : null,
+			},
+			isNull
+		),
+		<Gridicon icon="multiple-users" size={ props.size } className="presence-button__icon" />,
+		<span className="presence-button__label">
+			<span className="presence-button__label-count">{ presenceCount }</span>
+
+			{ showLabel && (
+				<span className="presence-button__label-status">
+					{ translate( 'Viewing', 'Viewing', {
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.